src/components/response/RadioInput.tsx

```diff
@@ -60,62 +60,64 @@ export function RadioInput({
     >
       <Group gap="lg" align="flex-end" mt={horizontal ? 0 : 'sm'}>
         {leftLabel ? <Text>{leftLabel}</Text> : null}
-        <HorizontalHandler horizontal={!!horizontal} style={{ flexGrow: 1 }}>
-          {orderedOptions.map((radio) => (
-            <div
-              key={`${radio.value}-${response.id}`}
-              style={{
-                display: 'flex',
-                flexDirection: horizontal ? 'column' : 'row',
-                gap: horizontal ? 'unset' : rem(12),
-                flex: stretch ? 1 : 'unset',
-                alignItems: 'center',
-              }}
-            >
-              {horizontal && <Text size="sm">{radio.label}</Text>}
-              <Radio
-                disabled={disabled}
-                value={radio.value}
-                label={radio.label}
-                styles={{
-                  label: { display: !horizontal ? 'initial' : 'none' },
+        <div className="radioButtons">
+          <HorizontalHandler horizontal={!!horizontal} style={{ flexGrow: 1, display: 'flex', justifyContent: 'space-between' }}>
```

Comment:
Is this style on the horizontal handler a hack to make the images line up? We should remove if the markdown image labels works.